src/test/java/org/matsim/run/drt/RunDrtOpenBerlinScenarioTest.java

```diff
@@ -104,6 +99,7 @@ public final void eTest0_1pctUntilIteration3() {
 	
 	// During debug some exceptions only occured at the replanning stage of the 3rd
 	// iteration, so we need at least 3 iterations.
+	@Ignore //ignored this test since we removed pt-extensions from pom.xml
```

Comment:
I would rather delete entirely these ignored tests

Reply:
ok, was not sure if you would still like to have them included